backend/migrations/2023-07-22-110000_shadings/down.sql

```diff
@@ -0,0 +1,28 @@
+-- This file should undo anything in `up.sql`
+DROP FUNCTION calculate_score_from_shadings;
+
+CREATE OR REPLACE FUNCTION calculate_score(
+    p_map_id INTEGER,
+    p_layer_ids INTEGER [],
+    p_plant_id INTEGER,
+    date DATE,
+    x_pos INTEGER,
+    y_pos INTEGER
+)
+RETURNS SCORE AS $$
+DECLARE
+    score SCORE;
```

Comment:
The same as in [backend/migrations/2023-07-03-165000_heatmap/up.sql](https://github.com/ElektraInitiative/PermaplanT/pull/684/files#diff-90e920f3ee01f0da92f3e2f47880632407ab1bba770c44355e9bd0c864877896).

Is there any reason for using the variable `score` instead of directly returning `plants`?